README.md

```diff
@@ -37,7 +37,7 @@ children below it will also be removed from the hierarchy.
 ```toml
 # Cargo.toml
 [dependencies]
-specs-hierarchy = "0.2.1"
+specs-hierarchy = "0.3.1"
```

Comment:
```suggestion
specs-hierarchy = "0.3.0"
```

Reply:
@torkleyy , I don't understand. "0.3.1" will get Cargo use specifically this version on crates.io.  "0.3" will get it to use any version with three as the middle number that it can find on crates.io. Wouldn't "0.3.0" force it to specifically look for the outdated version? Why is this desired?

Reply:
Oops, just noticed my comment is crap. I assumed you bumped the version to 0.3, but it turns out you only fixed the readme without bumping the version.

Please bump to 0.4.0 so I can release once this PR is merged.

Regarding version requirements: No, specifiyng `0.3.0` does not lock the version; it is equivalent to `^0.3.0`, which means "any version API-compatible with 0.3.0" (which 0.3.1 is). Only `=0.3.0` would lock.

Reply:
Alright, I'll bump to 4.0 and thanks, I didn't know that about the semantic versioning.